src/Divider/Divider.js

```diff
@@ -22,7 +23,12 @@ export const styleSheet = createStyleSheet('Divider', (theme) => {
   };
 });
 
-export default function Divider(props, context) {
+type Props = {
+  absolute?: boolean,
+  className?: string,
+};
+
+export default function Divider(props:Props, context:{styleManager: Object}):Element {
```

Comment:
I feel like in a lot of examples of typing languages, I see that it's common to add a space in between the argument and the type.

```js
Divider(props: Props, context: {styleManager: Object}): Element {
```

What do you think?